tests/test_human_data_cache.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import pytest
+
+from mmhuman3d.data.data_structures.human_data import HumanData
+from mmhuman3d.data.data_structures.human_data_cache import (
+    HumanDataCacheReader,
+    HumanDataCacheWriter,
+)
+
+human_data_cache_path = 'tests/data/human_data/human_data_cache.npz'
+
+
+def test_write():
+    human_data = HumanData.new(key_strict=False)
+    human_data['keypoints2d'] = np.ones(shape=(100, 199, 3))
+    human_data['keypoints2d_mask'] = np.ones(shape=(199, ))
+    human_data['keypoints2d_convention'] = 'human_data'
+    human_data['keypoints2d'][50, 50, :] *= 3
+    human_data['keypoints4d'] = np.ones(shape=(100, 144, 3))
```

Comment:
Why `keypoints4d` (not `3d`)?

Reply:
Because we are testing a key not in HumanData.SUPPORTED_KEYS here.